### ISSUE TITLE
Scripts assume the build is release

### ISSUE THREAD

**Issue by trufae** [fabricedesre/servonk#4]
Those platform/glutin/support/scripts/*.sh assume the build was done in release mode, it should check which one is the most recent (release|debug) and use that, or let the user specify it with an env var.

<img width="662" alt="screen shot 2018-04-18 at 10 06 21" src="https://user-images.githubusercontent.com/6431515/38919292-7b750e20-42f0-11e8-83d6-d818e55523a6.png">

**Comment by radare** [comment]
why assume release?

**Comment by fabricedesre** [comment]
Because there is no reason to use slower builds as default ones?

**Comment by radare** [comment]
default is debug unless you pass the —release flag

**Comment by fabricedesre** [comment]
Yeah, for the build script. We need to change that actually. Wanna fix that?

**Comment by radare** [comment]
yes i can send a PR for this, but not now gtg busy irl
